services/gitlab.service

```diff
@@ -18,7 +18,8 @@ ExecStartPre=/usr/bin/env bash -c "/usr/bin/docker run -d \
     --net protonet \
     --volume /data/gitlab/data:/home/git/data \
     -p \"$(/opt/bin/gitlab-network show):2222:22\" \
-    --env \"GITLAB_HOST=$(/opt/bin/gitlab-network show)\" \
+    --env \"GITLAB_HOST=gitlab.$(</etc/protonet/hostname).protonet.info\" \
+    --env \"GITLAB_SSH_HOST=$(/opt/bin/gitlab-network show)\" \
```

Comment:
Yes, this is one more of those things that will need automated configuration management, basically a single source of truth to ask for the current domain name that contains the logic instead of those tiny sprinkles here and there